deploy/vagrant/core/init.sh

```diff
@@ -26,7 +26,7 @@ then
 EOF
     # ssh config
     mkdir -p ~/.ssh
-    src="/tachyon/deploy/vagrant/files"
+    src="/vagrant/files"
```

Comment:
if version is github, then ../../deploy/vagrant is not synced to /tachyon, /tachyon is where the cloned repo is. but deploy/vagrant on host is always synced to /vagrant, and "files" can always be found in /vagrant/files in vagrant virtual machine.